index.html

```diff
@@ -4,6 +4,6 @@
     Git Rebasing Demo
     first commit
```

Comment:
```suggestion
    first commit added
```
add 'added' here also